AirCasting/Extensions/Date+Miliseconds.swift

```diff
@@ -7,7 +7,7 @@ extension Date {
     static let msFormatter: DateFormatter = DateFormatters.DateExtension.milisecondsDateFormatter
     
     var milliseconds: Int {
```

Comment:
please remove changes from this extension and add `swiftlint:disable` marks here too